source/_partial/network_stubbing_warning.md

```diff
@@ -1,5 +1,8 @@
-{% note danger %}
-🚨 Please be aware that Cypress only currently supports intercepting XMLHttpRequests. **Requests using the Fetch API and other types of network requests like page loads and `<script>` tags will not be intercepted or visible in the Command Log.** You can automatically polyfill `window.fetch` to spy on and stub requests by enabling an [experimental](https://on.cypress.io/experimental) feature `experimentalFetchPolyfill`. See {% issue 95 %} for more details and temporary workarounds.
+{% note warning %}
+⚠️ `cy.route()` and `cy.server()` only support intercepting XMLHttpRequests. Requests using the Fetch API and other types of network requests like page loads and `<script>` tags will not be intercepted by `cy.route()` and `cy.server()`.
 
-Cypress also has a new experimental [route2](/api/commands/route2.html) feature that supports requests using the Fetch API and other types of network requests like page loads. For more information, check out the [cy.route2() documentation](/api/commands/route2.html).{% endnote %}
+**To support requests using the Fetch API you can use one of the solutions below:**
 
+- Use [`cy.http()`](/api/commands/http.html) which supports requests using the Fetch API and other types of network requests like page loads. See [`cy.http()`](/api/commands/http.html).
+- Polyfill `window.fetch` to spy on and stub requests using `cy.route()` and `cy.server()` by enabling [`experimentalFetchPolyfill`](https://on.cypress.io/experimental). See {% issue 95 %} for more details and temporary workarounds.
+{% endnote %}
```

Comment:
With `cy.http()` becoming official, I believe these deprecation methods need to be more prominent. It seems odd to list multiple "workarounds" when there is an officially supported version. I suggest removing the `experimentalFetchPolyfill` and explain that `cy.http()` can now achieve this directly.

Reply:
yah, I agree